modules/kubernetes/pages/k8s-operator/installation.adoc

```diff
@@ -0,0 +1,43 @@
+= Installing TigerGraph Kubernetes Operator
+:description: Instructions on how to install TigerGraph Kubernetes Operator.
+:sectnums:
+
+This page describes the steps to install TigerGraph Kubernetes Operator on your Kubernetes cluster.
+
+The Operator only needs to be installed once for each cluster.
+If you run into the CRD
+
+== Prerequisites
+* You have configured the `kubectl` command-line tool for access to your running Kubernetes cluster.
+* Helm is installed on your local machine.
+* https://cert-manager.io/docs/installation/kubectl/[cert-manager is installed on your Kubernetes cluster].
+* If you are pulling from a private registry, configure your cluster for access to the registry.
+You can do this by https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret[creating a Secret] and https://kubernetes.io/docs/concepts/configuration/secret/#use-case-as-container-environment-variables[use it as a container environment variable].
+
+== Procedure
+
+[#_install_the_tigergraph_kubectl_plugin]
+=== Install the TigerGraph `kubectl` plugin
+Run the command below to install TigerGraph plugin for the `kubectl` commandline tool:
```

Comment:
Run the command below to install the TigerGraph plugin for the `kubectl` command line tool: